examples/platform/esp32/common/Esp32AppServer.cpp

```diff
@@ -114,6 +114,15 @@ void Esp32AppServer::DeInitBLEIfCommissioned(void)
 #endif /* CONFIG_USE_BLE_ONLY_FOR_COMMISSIONING */
 }
 
+void Esp32AppServer::Shutdown()
+{
+    chip::DeviceLayer::Internal::BLEMgr().Shutdown();
+#if CHIP_DEVICE_CONFIG_ENABLE_THREAD
+    chip::app::DnssdServer::Instance().StopServer();
+#endif
+    chip::Server::GetInstance().Shutdown();
```

Comment:
Also, did you try compilation with any of the Thread SoC

Reply:
Added thread shutdown flow.